### PR TITLE
[Docs] Installation instructions for GitPod

### LINKED ISSUE #9965: [Docs] Installation instructions for GitPod
### Current State

No docs instructions for running Meshery in GitPod

### Desired State

GitPod installation instructions for Meshery in Meshery Docs. It would be nice to have this documented alongside the instructions for running [Meshery in GitHub Codespaces](https://docs.meshery.io/installation/codespaces). 

### Task

1. Create a similar docs page for GitPod

### Context

> Hello everyone, so I tried running the site and the server on Gitpod and it was successfully setup and I think using Gitpod simulates a dev env very easily without having even `make` or `docker` installed on your local machine. So I want to add the part "accessing meshery server, ui, site using Gitpod" in the existing documentation. We can access the server, site etc in our localhost even by simply port forwarding (tunnel) from our machine to a remote host by using the following command--
> ```ssh -L 9081:localhost:9081 `your gitpod access token````
> 

[Slack Message](https://layer5io.slack.com/archives/CFGG6U10E/p1705246297639579?thread_ts=1705246297.639579&cid=CFGG6U10E)

### PR DESCRIPTION
**Pull request by MonalikaPatnaik**
**Notes for Reviewers**

This PR fixes #9965
Added the installation instructions for gitpod just same as [Meshery in GitHub Codespaces](https://docs.meshery.io/installation/codespaces), only the tool has been changed from codespaces to Gitpod.
Let me know if any further changes are required.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@ask-03, @ayushgml , would you like to review this?

Thanks, @MonalikaPatnaik. 👍

**Comment by ayushgml** [comment]
Yes @leecalcote. I would like to review this. Thanks for asking!

**Comment by ASK-03** [comment]
On it @leecalcote! Thanks for asking : )

**Review by ayushgml — CHANGES_REQUESTED** [review]
Hey @MonalikaPatnaik the content looks good to me just the Gitpod image is not show like below behaviour:
Screenshot 2024-01-20 at 2 19 44 PM

The likely cause for this is due to the mismatch of the image location in the ```gitpod.md``` file from the actual location like highlighted below:

Screenshot 2024-01-20 at 2 19 44 PM (2)


Place the image in ```docs/assets/img/platforms/``` instead of ```docs/assets/img/``` and it should work fine.

File: docs/pages/installation/gitpod.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+**To allow Meshery to auto detect your config file, Run** :
+{% capture code_content %}kubectl config view --minify --flatten > config_minikube.yaml{% endcapture %}
+{% include code.html code=code_content %}
+
+<br />
+Meshery should now be connected with your managed Kubernetes instance. Take a look at the [Meshery guides]({{ site.baseurl }}/guides) for advanced usage tips.
+
+## Access Meshery UI
+
+To access Meshery's UI, please refer to the [instruction](/project/contributing/contributing-ui#setup) for detailed guidance.
```
Comment (ayushgml) [inline docs/pages/installation/gitpod.md:128]: Change the link to: ```[instruction](/tasks/accessing-meshery-ui)```

File: docs/pages/installation/gitpod.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+You can develop and run Meshery in a Gitpod using your choice of tool:
+
+- A command shell, via an SSH connection initiated using GitHub CLI.
+- One of the JetBrains IDEs, via the JetBrains Gateway.
+- The Visual Studio Code desktop application.
+- A browser-based version of Visual Studio Code.
+#### Fork and Open in Gitpod:
+- Open your forked Meshery repository on GitHub.
+- Click on the "Gitpod" button in the top right corner of the repository page.
```
Comment (ayushgml) [inline docs/pages/installation/gitpod.md:56]: Mention that the Gitpod button is only visible with the browser extension

**Review by ayushgml — CHANGES_REQUESTED** [review]
A few more changes

File: docs/pages/installation/gitpod.md (hunk 2 above)
Comment (ASK-03) [reply docs/pages/installation/gitpod.md:56]: Hey @MonalikaPatnaik! You can also add the [link](https://chromewebstore.google.com/detail/gitpod/dodmmooeoklaejobgleioelladacbeki) to extension.

Link: https://chromewebstore.google.com/detail/gitpod/dodmmooeoklaejobgleioelladacbeki

Comment (MonalikaPatnaik) [reply docs/pages/installation/gitpod.md:56]: I have added to add the chrome extension for gitpod in the prerequisite only

**Comment by l5io** [comment]
🚀 Preview for commit 64872acb2ceaa7e376ef986379acfc9fdc800b90 at: https://65abc0c4e8f5d68f9f5950e4--meshery-docs-preview.netlify.app

**Review by ayushgml — APPROVED** [review]
Changes approved. Looks good to me @leecalcote!

**Comment by MonalikaPatnaik** [comment]
Thanks for the review @ayushgml @ASK-03 , I want to ask how to add the gitpod installation instructions in the left navigation panel of the docs unlike for codespaces.
image
but for gitpod there is no such thing
image

**Comment by ASK-03** [comment]
@MonalikaPatnaik! I've located the file that handles the rendering of the side panel, which is `docs/_includes/sidebar.html`. I am also looking for cause of this issue. Hope this helps : )

**Comment by MonalikaPatnaik** [comment]
> @MonalikaPatnaik! I've located the file that handles the rendering of the side panel, which is `docs/_includes/sidebar.html`. I am also looking for cause of this issue. Hope this helps : )

It's done now!

**Comment by l5io** [comment]
🚀 Preview for commit 1cc6f32b53bfbdb15f0b5f49002a46527e4ab0a0 at: https://65ac18810c7751d422244880--meshery-docs-preview.netlify.app

File: docs/pages/installation/gitpod.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+Once configured, execute the following command to start Meshery.
+
+{% capture code_content %}$ mesheryctl system start{% endcapture %}
+{% include code.html code=code_content %}
+
+If you encounter any authentication issues, you can use `mesheryctl system login`. For more information, click [here](/guides/mesheryctl/authenticate-with-meshery-via-cli) to learn more.
+
+## Installation: Using Helm
+
+For detailed instructions on installing Meshery using Helm V3, please refer to the [Helm Installation](/installation/helm) guide.
```
Comment (ASK-03) [inline docs/pages/installation/gitpod.md:86]: @MonalikaPatnaik! This should be /installation/kubernetes/helm; currently, it is showing a 404 error. Could you please update it for CodeSpaces as well? It's incorrect there too

**Comment by l5io** [comment]
🚀 Preview for commit 053fafa808bca53c608ae887a8189e4ccf470c78 at: https://65acca4d865d583056fce5da--meshery-docs-preview.netlify.app

**Comment by ASK-03** [comment]
LGTM! @leecalcote. Thank you for your contribution @MonalikaPatnaik! 🎉

**Comment by ayushgml** [comment]
Thank you for your contribution @MonalikaPatnaik!

**Comment by saurabh100ni** [comment]
Hi @MonalikaPatnaik  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Comment by l5io** [comment]
🚀 Preview for commit a1d72f621a4a7c933a7ce5ea2bd0c5fe1a7d2739 at: https://65ad437b70a8277b82649f25--meshery-docs-preview.netlify.app

**Comment by sudhanshutech** [comment]
@MonalikaPatnaik as discussed in meet, you have the following documentation like https://docs.meshery.io/project/contributing/contributing-docs#using-docker here since as Lee also said gitpod for now is uncertain in this conditions so in case of contributing it looks more appropriate. Make sense?

**Comment by MonalikaPatnaik** [comment]
> @MonalikaPatnaik as discussed in meet, you have the following documentation like https://docs.meshery.io/project/contributing/contributing-docs#using-docker here since as Lee also said gitpod for now is uncertain in this conditions so in case of contributing it looks more appropriate. Make sense?

Docker is best I understand, but using it in windows is a bit difficult as compared to Gitpod. Gitpod  comes with Docker and other tools pre-installed making it one of the fastest ways to set up our dev environment with which we don't need to install make, docker, WSL etc on our system globally.

**Comment by l5io** [comment]
🚀 Preview for commit e547bca53dab36f20edc2efc57710ef379c7fc9f at: https://65bcf245768bad8eb8750413--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 098c0a674bc0691dfb734d64241d1745b4124fbb at: https://65bd53213dab3302bade94ea--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 86a99841b39a108e1784cfe6da5cbc5e44d471d0 at: https://65bd54b73c6c59030cc6565d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 014f891564e8f2dd6f9c08081d6bef32b293ba0e at: https://65be437599ee79927e7e9959--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 294900d37b7228ddcb324118b35712fd3c3b1326 at: https://65be44e2b90990960d794a68--meshery-docs-preview.netlify.app

**Pull request closed without merging**